frontend/components/MostViewed.tsx

```diff
@@ -151,30 +152,21 @@ interface Trail {
     linkText: string;
 }
 
-export default class MostViewed extends Component<{}, { trails: Trail[] }> {
-    constructor(props: {}) {
-        super(props);
-        this.state = {
-            trails: [],
-        };
-    }
-
-    public componentDidMount() {
-        fetch('https://api.nextgen.guardianapps.co.uk/most-read-geo.json?guui')
-            .then(resp => resp.json())
-            .then(({ trails }) => {
-                this.setState({
-                    trails,
-                });
-            });
-    }
-
-    public render() {
-        return (
-            <div className={container}>
-                <h2 className={heading}>Most Viewed</h2>
-                <ul className={list}>
-                    {this.state.trails.map((trail, i) => (
+const fetchTrails = async () => {
+    const resp = await fetch(
+        'https://api.nextgen.guardianapps.co.uk/most-read-geo.json?guui',
+    );
+    const j = await resp.json();
```

Comment:
We should add a check to see whether the response is ok here `if (resp.ok) {}` and gracefully handle responses that aren't ok, once we have error recording in place we should record these occurences. For now maybe we just resolve with an empty array?